app/scripts/controllers/app-state.js

```diff
@@ -53,6 +54,7 @@ export default class AppStateController extends EventEmitter {
       trezorModel: null,
       currentPopupId: undefined,
       onboardingDate: null,
+      newPrivacyPolicyToastDate: PRIVACY_POLICY_DATE,
```

Comment:
A bunch of touched files. But this moves the privacy policy into app state - which allows us to use fixtures and mocks for our e2e and unit tests